### ISSUE TITLE
[Feature request] Queue commands after new line character

### ISSUE THREAD

**Issue by vlad2305m** [chirs241097/jline4mcdsrv#15]
```
/
> /say hi
> /say thanks for playing
[22:12:49] [Server thread/INFO] (MinecraftServer) <Server> hi
/say thanks for playing
 ```
Currently exectutes the whole string. ([Server] hi\n/say thanks for playing)
This is a very nice behavior, but new line character is kindof useless in MC, as it displays as 'LF' outside of chat
It would be nice to use it to queue commands instead, and then escape it each time it is actually needed.

And, of course, autocompletion for each line.

Reproduce: Ctrl+V -> Ctrl+J on Linux, or paste smth from notepad on Win

**Comment by Fourmisain** [comment]
That is unexpected,
https://github.com/chirs241097/jline4mcdsrv/blob/fb295a32866e97f7e964bc212cf0c8ea64c48def/src/main/java/org/chrisoft/jline4mcdsrv/Console.java#L60-L63
can actually read multi-line commands, despite `readLine`'s name.

I agree that it should execute each line as it's own thing, since that's "vanilla" behavior (of any basic console on the planet too).

While it's theoretically possible to keep multi-lines _only_ for chat commands like `/say`, `/msg` etc, this seems really janky to implement and ambiguous too. Commands can omit the `/`, so it's not clear if something like
```
/say Didn't mean to do that,
pardon me
```
should print 2 lines of text or print 1 line and unban the player named "me".

Multi-line chats can always be done via `/tellraw`, e.g. your example ould turn into
`/tellraw @e[type=player] {"text":"hi\nthanks for playing"}`
and since this only applies to copy-pasted text (at least I don't know how else to type newline characters), I think it'd be best to kick multi-line commands out and simply treat each line separately.

---

I did a little bit of work on that now, the main thing of handling each line separately (acddcb1616f1dc88d0806bce75092936622f0a05) and the syntax highlighting f9b79e27324fbcf2e8a121465772fbd942ddaebc is done and as for the auto-completion, it might not even be possible to do that? I'm not sure.

I'll look over this another time and hopefully do a release then; in the meantime you can build the repo and do some testing if you want!

**Comment by vlad2305m** [comment]
Hi @Fourmisain, 
Just wanted to say I'm on it. (Took some trickery to set up intellij debugging with cmd.exe)
```
/say it is so nice
/say that suggestions got fixed
/say it even started highlighting syntax on its own
/particle minecraft:firework 0 100 0 1 2 3 1 1
[13:58:58] [Server thread/INFO] (MinecraftServer) [Not Secure] [Server] it is so nice
[13:58:58] [Server thread/INFO] (MinecraftServer) [Not Secure] [Server] that suggestions got fixed
[13:58:58] [Server thread/INFO] (MinecraftServer) [Not Secure] [Server] it even started highlighting syntax on its own
[13:58:58] [Server thread/INFO] (MinecraftServer) The particle was not visible for anybody
/say
/save-on[caret]
/say
save-all   save-off   save-on    say
```
---
~~Btw, you broke something:~~ Fixed
When inputting \n first (`Ctrl+V` `Ctrl+J`)

**Comment by vlad2305m** [comment]
#16 
Thank you very much for helping me with another mod.
Do you happen to know where I can find docs on Mixin Plugins (apply if not [mod] present) ?

**Comment by Fourmisain** [comment]
Thanks for the PR! Will have a look at it tomorrow (probably).

> Do you happen to know where I can find docs on Mixin Plugins (apply if not [mod] present) ?

While the [Mixin wiki](https://github.com/SpongePowered/Mixin/wiki) doesn't cover that yet, what you're searching for can be found in the [Mixin javadocs](https://jenkins.liteloader.com/view/Other/job/Mixin/javadoc/overview-summary.html), specifically for the [`IMixinConfigPlugin`](https://jenkins.liteloader.com/view/Other/job/Mixin/javadoc/org/spongepowered/asm/mixin/extensibility/IMixinConfigPlugin.html).

The short version of how to use it is:

 - create a class implementing the `IMixinConfigPlugin` interface and let `shouldApplyMixin` return false for the given `mixinClassName` when the mod is(n't) present (using `FabricLoader.getInstance().isModLoaded(modId)`)
 - add the plugin to your `yourmod.mixins.json` like `"plugin": "your.package.YourPlugin",`

**Comment by vlad2305m** [comment]
Thank you very much!
Please add the following somewhere to the ReadMe:
```
How to debug in IntelliJ while running in cmd:
1. Fork and setup like any other Minecraft mod
2. Run Server configuration, accept EULA, make it work
3. Create debug.bat, paste the first line from the Server run log into there, append '\npause' if you want to see crashes (the one that expands to oblivion when clicked ("C:..."))
4. Edit Configurations... / Add New Configuration / Remote JVM Debug -> Transport=Shared Memory, Address=localhost:25555 (not tested other values, but address should be filled), Use module classpath=main.
5. Copy Command line arguments for remote JVM and paste into the beginning of debug.bat replacing existing agentlib config
6. Don't forget to add `-Dlog4j.skipJansi=false` to script and save it
Workloop: Build -> Run debug.bat -> launch debugger -> debug normally -> close cmd manually -> repeat
Note: Each time launching Minecraft Server configuration changes the first line a bit, so you might need to update stuff after agentlib config in the script. It looks monstrous (13K symbols), so I don't know what changes exactly. Definetely update after adding new files.
Any further attempts to automate result in IntelliJ consuming the cmd window and blocking the suggestions/highlighting
```
